EverOddish/OpenFuji
# General cleanup of Drfujibot

Summary:

First general step in cleaning up DrFujibot and
making the code more pythonic.

- Remove an unneeded package from requirements.txt

- Cleanup drfujibot_pykemon/requests.py in accordance with PEP8
	styleguide

- Change BASE_URI to point to 'https://pokeapi.co/api/v2' for
	easier setup for newer contributors.
	If you have a local version of the pokeapi running
	you will need to revert this particular change
	if you want to keep using it, also in some instances
	in drfujibot.py
	In future should add BASE_URI as an option so you will
	only need to change uri once depending on what you want
	to use.

- Update readme to give more detailed instructions on setup of
	drfujibot and how to test the bot

- Add sql files to .gitignore because drfujibot_pykemon/request.py
	will generate sql files

- fujibot config file to gitignore and give instructions for making
	a config file and filling it out in readme

- drfujibot.py
	- Sort imports according to PEP8 styleguide
	- add Exception to  all instances of empty except statement
	- Clean up a lot of if/else statements to be more pythonic
	- Make sure all lines are shorter than 80 characters
		according to PEP8 styleguide
	- Add a few comments about unused variables for review
